server/routes/testutils/testData.ts

```diff
@@ -354,15 +355,16 @@ export default class TestData {
   static supportedPrisonIds = ({ prisonIds = ['HEI', 'BLI'] } = {}): string[] => prisonIds
 
   static prison = ({
-    prisonId = 'HEI',
-    prisonName = 'Hewell (HMP)',
-    active = true,
-    policyNoticeDaysMax = 28,
-    policyNoticeDaysMin = 2,
-    maxTotalVisitors = 6,
-    maxAdultVisitors = 3,
-    maxChildVisitors = 4,
-    adultAgeYears = 18,
+    prisonId = this.prisonDto().code,
```

Comment:
`Prison` type is effectively a clone of raw `PrisonDto` from API so keeping default test data values consistent between the two.